src/components/ContactUsPage/ContactUsPage.js

```diff
@@ -2,168 +2,33 @@ import React, { useEffect, useState } from 'react';
   const [mainContent, setMainContent] = useState();
-  const [commentTypes, setCommentTypes] = useState([]);
-  const [submitStatusText, setSubmitStatusText] = useState([]);
-
-  const [submitted, setSubmitted] = useState(false);
-  const [submitStatus, setSubmitStatus] = useState(false);
-  const [emailErrorMsg, setEmailErrorMsg] = useState('');
 
   useEffect(() => {
     document.title = 'Contact Us | CAMPD | US EPA';
-
-    // This is done to have the page structure 508 compliant
-    const h3Tag = document.querySelector('h3');
-    if (h3Tag) h3Tag.outerHTML = '';
   }, []);
 
-  useEffect(() => {
-    const usaAlert = document.querySelector('.usa-alert');
-    if (usaAlert) {
-      window.scrollTo(0, document.body.scrollHeight);
-      usaAlert.setAttribute('tabIndex', 0);
-      usaAlert.focus();
-
-      const h4Tag = document.querySelector('h4');
-      if (h4Tag) {
-        if (h4Tag) h4Tag.outerHTML = `<h2> ${h4Tag.innerHTML} </h2>`;
-      } else {
-        const h2Tag = document.querySelector('h2');
-        if (h2Tag) {h2Tag.outerHTML = `<h2> ${submitStatus ? 'Success' : 'Error'} </h2>`};
-      }
-    }
-  }, [submitted, submitStatus]);
-
   metaAdder(
     'description',
-    'Utilize the Contact us page to submit a help ticket to the Clean Air Markets Division'
+    'Contact the Clean Air Markets Division for support with CAMPD'
   );
-  metaAdder('keywords', 'CAMPD, CAMD, help, contact, support, ticket');
+  metaAdder('keywords', 'CAMPD, CAMD, help, contact, support');
 
   useEffect(() => {
     getContent('/campd/help-support/contact-us/index.md', setApiErrorDispatcher).then((resp) =>
     resp && setMainContent(resp.data)
-    );
-
-    getContent('/campd/help-support/contact-us/comment-types.json', setApiErrorDispatcher).then(
-      (resp) => resp && setCommentTypes(resp.data)
-    );
-
-    getContent('/campd/help-support/contact-us/submit-status-text.json', setApiErrorDispatcher).then(
-      (resp) => {
-        if (resp){
-          const modifiedStatusObject = resp.data.map((status) => {
-            if (status.hasOwnProperty('email')) {
-              const splitMessage = status.message.split('[email]');
-              return {
-                status: status.status,
-                message: (
-                  <span>
-                    {splitMessage[0]}
-                    <USWDSLink
-                      to="#"
-                      onClick={(e) => {
-                        window.location = `mailto:${status.email}`;
-                        e.preventDefault();
-                      }}
-                    >
-                      {status.email}
-                    </USWDSLink>
-                    {splitMessage[1]}
-                  </span>
-                ),
-              };
-            } else {
-              return status;
-            }
-          });
-          setSubmitStatusText(modifiedStatusObject);
-        }
-      }
     );//eslint-disable-next-line
```

Comment:
Sonarcloud Quality Gate failed 
```suggestion
    );// eslint-disable-next-line react-hooks/exhaustive-deps
```

Reply:
Resolved by properly adding setApiERrorDispatcher to the dependency array